moped-editor/src/utils/activityLogFormatters/mopedProjectActivity.js

```diff
@@ -0,0 +1,59 @@
+import BeenhereOutlinedIcon from "@material-ui/icons/BeenhereOutlined";
+import { ProjectActivityLogTableMaps } from "../../views/projects/projectView/ProjectActivityLogTableMaps";
+
+export const formatProjectActivity = (change, entityList) => {
+  const entryMap = ProjectActivityLogTableMaps["moped_project"];
```

Comment:
the casing on `ProjectActivityLogTableMaps ` is throwing me off because i'm expecting this to be a class. exporting a name like `TABLE_MAPS` would be better? i see the pattern throughout this feature so nbd to leave this 🧹 for a future PR.

Reply:
This has been around forever, I'll change the name in a future PR